src/runtime/server/middleware/headers.ts

```diff
@@ -0,0 +1,13 @@
+import { defineEventHandler, setHeader, removeResponseHeader } from '#imports'
+
+export default defineEventHandler((event) => {
+    if(event.context.security.headers) {  
+        Object.entries(event.context.security.headers).forEach(([header, value]) => {
+            if(value === false) {
+                removeResponseHeader(event, header)
+            }else {
+                setHeader(event, header, value, )
```

Comment:
question: is this correct? There is `value, ` at the end.

Reply:
oops, i need to lint that